src/main/java/org/veupathdb/eda/binaryfiles/dumper/Main.java

```diff
@@ -64,4 +74,32 @@ public static void main(String[] args) throws Exception {
       studyDumper.dumpStudy();
     }
   }
+
+  public static Map<String, String> parseConfig(Path filePath) {
+    Map<String, String> config = new HashMap<>();
+
+    try (BufferedReader br = Files.newBufferedReader(filePath)) {
+      String line;
+      while ((line = br.readLine()) != null) {
+        if(line.startsWith("#")) {
+          continue;
+        }
+
+        String[] ar = line.split("=");
+        if (ar.length == 2) {
+          config.put(ar[0], ar[1]);
+        }
+      }
+    }
+    catch (NoSuchFileException e) {
+      throw new RuntimeException("File not found: " + filePath);
+    }
+    catch (IOException e) {
+      e.printStackTrace();
+      throw new RuntimeException("Error Reading File:" + filePath);
```

Comment:
You can get rid of `e.printStackTrace()` and just attach the exception as the cause of the RuntimeException (i.e. `throw new RuntimeException("Error Reading File:" + filePath, e);` and it will get printed to stdout.